playground/mocks/spec-okta-api/idp/idx/index.js

```diff
@@ -25,7 +25,8 @@ const idx = [
   '/idp/idx/recover',
   '/idp/idx/skip',
   '/idp/idx/poll',
-  '/idp/idx/unlock-account'
+  '/idp/idx/unlock-account',
+  '/idp/idx/request-activation'
```

Comment:
can you add a trailing comma here so the next person to add to this list doesn't trigger a change for the line above?